README.md

```diff
@@ -177,7 +177,7 @@ export default class FlashMessages {
 - visit(url)
 - currentUrl()
 - getParam(param)
-- setParam(param, value)
+- setParam(param, value, url = currentUrl())
```

Comment:
I'd delete the default value detail for now... In the future we need to document properly each method